realm/realm-library/src/androidTest/java/io/realm/DynamicRealmTest.java

```diff
@@ -882,4 +886,42 @@ public void run() {
         }
     }
 
+    public void testClearAll() {
+        realm.beginTransaction();
+        realm.createObject(CLASS_ALL_TYPES);
```

Comment:
I would move these constants to their class instead, like "Alltypes.CLASS_NAME", some of them already exists.